src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -153,6 +177,15 @@ class PalantirDockerPlugin implements Plugin<Project> {
         }
     }
 
+    private List<String> loginCommandline(DockerLoginExtension loginExt){
+        List<String> loginCommandLine = ['docker', 'login']
+        if(loginExt.username && loginExt.password){
+            loginCommandLine.addAll('-u', "${-> loginExt.username}", '-p', "${-> loginExt.password}")
```

Comment:
This isn't safe because if you run `./gradlew dockerLogin -i` (e.g. on CI) then the password will be logged to the console!

I ended up doing a `sh -c "docker login -u $USERNAME -p $PASSWORD"` where the values are passed in to the environment.

Reply:
I am not sure what you mean. Do you suggest the user should set the values $USERNAME and $PASSWORD as environment variables or is there a possibility that the plug in can pass these to to environment of the exec tasks?